x-pack/plugins/observability/public/typings/index.ts

```diff
@@ -6,3 +6,4 @@
 
 export * from './eui_draggable';
 export * from './eui_styled_components';
+export * from './fetch_data_response';
```

Comment:
@jasonrhodes That looks good to me, I'm just curious to understand why the current way doesn't work for you. I am importing it on APM, and it works. https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/services/rest/observability_dashboard.ts#L11

Reply:
Interesting, @cauemarcondes -- I'm surprised you're not getting a linting error there. Just 2 lines above that you have the "// eslint-disable-next-line @kbn/eslint/no-restricted-paths" comment which is exactly about this same thing. My understanding is you can only import from plugin/public or plugin/server when importing from another plugin.

Reply:
Interesting, @cauemarcondes -- I'm surprised you're not getting a linting error there. 2 lines above you have "// eslint-disable-next-line @kbn/eslint/no-restricted-paths" which is about this very thing. My understanding is you can only import from plugin/public or plugin/server, when importing from other plugins.

Reply:
For the record, I had to rename `fetch_data_response/index.d.ts` to `index.ts`. Otherwise webpack complained when trying to create the bundle